centinel/client.py

```diff
@@ -123,8 +125,46 @@ def run(self, data_dir=None):
 
             try:
                 logging.info("Running %s test." % (name))
+
+                root = True
+                if os.geteuid() != 0:
+                    logging.info("Centinel is not running as root, "
+                                 "tcpdump will not start.")
+                    root = False
```

Comment:
To avoid having to check the root variable later, you should exit/return as soon as you know that you aren't root. Basically, it lets you remove 138 and remove a level of indentation